### PR TITLE
Modified the tests to use TestRepository annotation

### PR DESCRIPTION
**Pull request by vikss**
#1205 .

### CODE REVIEW

File: strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/maven/MavenMetadataManagementControllerTest.java [hunk 1]
```diff
@@ -1,57 +1,49 @@
...
 @IntegrationTest
 public class MavenMetadataManagementControllerTest
         extends MavenRestAssuredBaseTest
 {
 
-    private static final String REPOSITORY_RELEASES = "mmct-releases";
+    private static final String R1 = "mmct-releases";
 
-    private static final String REPOSITORY_SNAPSHOTS = "mmct-snapshots";
+    private static final String R2 = "mmct-snapshots";
```
Comment (carlspring) [inline strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/maven/MavenMetadataManagementControllerTest.java]: What's the reason for the renaming of this variable?

File: strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/maven/MavenMetadataManagementControllerTest.java [hunk 2]
```diff
@@ -60,27 +52,19 @@ public void init()
...
-                                       "3.0.2",
-                                       "3.1",
-                                       "3.2" });
+	@ExtendWith({ RepositoryManagementTestExecutionListener.class})
+    @Test
+    public void testRebuildSnapshotMetadata(@TestRepository(repositoryId = R2, layout = MavenArtifactCoordinates.LAYOUT_NAME, policy = RepositoryPolicyEnum.SNAPSHOT) Repository repository)
+            throws Exception
+    {
+		String repositoryId = repository.getId();
+        String metadataPath = "/storages/" + STORAGE0 + "/" + R2 +
```
Comment (carlspring) [inline strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/maven/MavenMetadataManagementControllerTest.java]: Cosmetic: Could you please follow the project's coding convention, as described in the wiki? Thanks! :)

File: strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/maven/MavenMetadataManagementControllerTest.java [hunk 3]
```diff
@@ -144,41 +106,43 @@ public void testRebuildSnapshotMetadata()
                .then()
                .statusCode(HttpStatus.OK.value());
 
+        //artifactMetadataService.
```
Comment (carlspring) [inline strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/maven/MavenMetadataManagementControllerTest.java]: Cosmetic: Could you please remove this commented out line?

Comment (vikss) [reply strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/maven/MavenMetadataManagementControllerTest.java]: Done.

File: strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/maven/MavenMetadataManagementControllerTest.java (hunk 2 above)
Comment (vikss) [reply strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/maven/MavenMetadataManagementControllerTest.java]: Done.

File: strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/maven/MavenMetadataManagementControllerTest.java (hunk 1 above)
Comment (vikss) [reply strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/maven/MavenMetadataManagementControllerTest.java]: In hindsight, there was no apparent reason. I have reverted this change.

**Comment by vikss** [comment]
@carlspring
This is ready for review again.

**Comment by carlspring** [comment]
Hi!

Thank you for your contribution! Does this solve the bug, or is it just covering the part with the refactoring?

Would you mind signing the ICLA, as described in the [Contributing](https://github.com/strongbox/strongbox/blob/master/CONTRIBUTING.md) page?

Kind regards,

Martin

**Comment by vikss** [comment]
> Hi!
> 
> Thank you for your contribution! Does this solve the bug, or is it just covering the part with the refactoring?
> 
> Would you mind signing the ICLA, as described in the [Contributing](https://github.com/strongbox/strongbox/blob/master/CONTRIBUTING.md) page?
> 
> Kind regards,
> 
> Martin

**Comment by vikss** [comment]
I couldn't reproduce the bug. So, it's just plain refactoring.

**Comment by vikss** [comment]
@carlspring 
Would you please review and merge it?

**Comment by carlspring** [comment]
@vikss ,

I can, but could you please sign the [ICLA](https://github.com/strongbox/strongbox/blob/master/CONTRIBUTING.md) first?

Thanks! :)

**Comment by carlspring** [comment]
Could you please rebase this to the latest code from our `master`? The tests aren't passing and we can't merge this until it has been resolved.

**Pull request closed without merging**